### ISSUE TITLE
Replace channel to ping from name to it's ID in the configuration file

### ISSUE THREAD

**Issue by 1anakin20** [1anakin20/ComputerSquadBot#37]
As the channel name can change, it's better to set the channel to send the ping alarms in by it's ID.

**Comment by zuliatowoade** [comment]
@1anakin20 I will like to work on this issue. Do you mind assigning it to me? Thanks

**Comment by 1anakin20** [comment]
Yes! I'll assign you on this issue, Feel free to ask me for any assistance you need.

**Comment by 1anakin20** [comment]
Fixed in #46

**Comment by zuliatowoade** [comment]
@1anakin20 per your comment, looks like this have been fixed in another PR. Was already working on it locally...I'll keep an eye out for another issue since I already have the app set up locally.

**Comment by 1anakin20** [comment]
Yes, I fixed it on another PR to keep it up to date. More issues may open soon, but as always feel free to suggest any changes.
